sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerAsyncClient.java

```diff
@@ -436,6 +436,26 @@ public String getEncryptionScope() {
             .map(response -> new SimpleResponse<>(response, null));
     }
 
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Void> createIfNotExists() {
+        try {
```

Comment:
Let's move this try/catch into the next overload so the error handling happens in a deeper level

Reply:
Check all other min overloads for the same thing